backoffice/views.py

```diff
@@ -305,3 +305,32 @@ def vehicles(request):
     context["page_range"] = page_range
 
     return render(request, "backoffice/vehicles.html", context)
+
+
+@require_backoffice_access
+def close_booking(request, booking_id, should_lock=True):
+    context = {
+        "menu": "bookings",
+        "user": request.user,
+    }
+
+    booking = Booking.objects.get(pk=booking_id)
+    booking.state = Booking.STATE_ENDED
```

Comment:
```suggestion
    booking.state = Booking.STATE_INACTIVE
```
The reason for this is it should match what happens when a user closes a booking normally - it doesn't go straight to `STATE_ENDED` - that's done by a cron job later.